backend/custom_models/migrations/0005_upload_emib_sample.py

```diff
@@ -0,0 +1,361 @@
+# Generated by Django 2.1.7 on 2019-05-30 14:07
+# Edited by J. Michael Cherry to populate the emib sample
+
+from django.db import migrations
+
+
+def upload_emib_sample(apps, schema_editor):
+    # get models
+    language = apps.get_model("custom_models", "Language")
+    item_type = apps.get_model("custom_models", "ItemType")
+    question_type = apps.get_model("custom_models", "QuestionType")
+    item = apps.get_model("custom_models", "Item")
+    item_text = apps.get_model("custom_models", "ItemText")
+    question = apps.get_model("custom_models", "Question")
+    # get db alias
+    db_alias = schema_editor.connection.alias
+    # create languages; do not use bulk_create since we need these objects later on
+    l_english = language(ISO_Code_1="en", ISO_Code_2="en-ca")
+    l_english.save()
+    l_french = language(ISO_Code_1="fr", ISO_Code_2="fr-ca")
+    l_french.save()
+
+    # create item_types; do not use bulk_create since we need these objects later on
+    it_test = item_type(type_desc="test")
```

Comment:
I like that we load these items types, because I think we'll also want them in the application no matter what for all other tests.